monai/networks/layers/simplelayers.py

```diff
@@ -434,6 +435,109 @@ def forward(self, x: torch.Tensor) -> torch.Tensor:
         return torch.as_tensor(ht, device=ht.device, dtype=ht.dtype)
 
 
+def get_binary_kernel3d(window_size: Sequence[int]) -> torch.Tensor:
+    r"""Create a binary kernel to extract the patches.
+    If the window size is HxWxD will create a (H*W*D)xHxWxD kernel.
+    """
+    window_range: int = window_size[0] * window_size[1] * window_size[2]
+    kernel: torch.Tensor = torch.zeros(window_range, 1, window_size[0], window_size[1], window_size[2])
+    for i in range(window_size[0]):
+        for j in range(window_size[1]):
+            for k in range(window_size[2]):
+                l: int = i * window_size[2] * window_size[1] + j * window_size[2] + k
+                kernel[l, 0, i, j, k] += 1.0
+    return kernel
+
+
+def _compute_zero_padding(kernel_size: Sequence[int]) -> List[int]:
```

Comment:
Is this doing the same as `same_padding` ?

Reply:
It turns out it is. The latest commit makes the change.